templates/components/amp/common/store-logo.html

```diff
@@ -2,8 +2,13 @@
     {{#if settings.store_logo.image}}
         <amp-img
             src="{{getImage settings.store_logo.image 'logo_size'}}"
-            width="100"
-            height="50"
+            {{#if theme_settings.logo_size '===' 'original' }}
+                width="250"
```

Comment:
I understand that in this case we need to hard code some values because the actual image size is unknown. How do we choose the right values? Why was this changed to 250 from 100?